package.json

```diff
@@ -47,7 +47,7 @@
   "license": "MIT",
   "main": "lib/index.js",
   "peerDependencies": {
-    "react": "^15.6.1"
+    "react": " ^15.0.0 || ^16.0.0"
```

Comment:
Q: Can you explain what you can translate this to?

Reply:
As `ui-icons` is merely a helper package for our other apps and is not really using React, it's a good practice to add React as a peerDependency. 

We are in the transitioning phase between React v15 and v16 (some projects still use 15), so we are adding both as a peerDep here so it removes the warnings when installing `ui-icons` if you had React v16 already.

 :ok: by @nickwaelkens on 15/11/2017 13:36:19